subjects/mobile-dev/map-markers/README.md

```diff
@@ -1,38 +1,67 @@
 ## map-markers
 
-### Introduction
+In this project, your team will create an app that uses the `Google Maps API`. The app must allow users to save and display their favorite places on a map.
 
-Create an app where you can save the list of your favorite places and show them in the app.
+### Instructions
 
-In this raid, your team should implement an app which uses google maps.
+The app should have the following functionality:
 
-### Objective
+A `TabBar` with three screens:
```

Comment:
In the audit, it is asked to look for a `Bottom TabBar` while here is "simply" `TapBar`

Reply:
done